src/windows/list.cpp

```diff
@@ -148,21 +148,26 @@ bool IsVirtualHardDrive(HDEVINFO hDeviceInfo, SP_DEVINFO_DATA deviceInfoData) {
   return false;
 }
 
-bool IsSystemDevice(HDEVINFO hDeviceInfo, SP_DEVINFO_DATA deviceInfoData) {
+bool IsSystemDevice(HDEVINFO hDeviceInfo, DeviceDescriptor *device) {
   PWSTR windowsPath = NULL;
+  BOOL isSystem = false;
   HRESULT result = SHGetKnownFolderPath(
     FOLDERID_Windows, 0, NULL, &windowsPath);
 
   if (result == S_OK) {
     std::string systemPath = WCharToUtf8(windowsPath);
-    CoTaskMemFree(windowsPath);
-    // printf("systemPath %s\n", systemPath.c_str());
-    // TODO(jhermsmeier): Compare against mountpoints to actually determine this
-    return true;
+    printf("systemPath %s\n", systemPath.c_str());
```

Comment:
Debugging prints, which ought to be commented out? ;)

Reply:
Silenced!